token-swap/proposals/StakePoolStableCurve.md

```diff
@@ -0,0 +1,9 @@
+# Combine Stable Curve with Stake Pool for price discovery
+
+Implement a curve for token-swap that accepts stake pool account and uses stable curve to swap assets. 
+[Stake pool account](https://github.com/solana-labs/solana-program-library/blob/master/stake-pool/program/src/state.rs#L17) will be used for initial price discovery then stable curve would be used to derive price based on the size of the order.
```

Comment:
tiny nit: stable curve would be used to derive price based on the size of the order and available reserves

is that a little clearer?